modules/functions.js

```diff
@@ -57,11 +57,14 @@ module.exports = (client) => {
   and stringifies objects!
   This is mostly only used by the Eval and Exec commands.
   */
+  
+  const util = require("util");
```

Comment:
Why is this being changed?

Reply:
If you mean `var` to `const`, for using less memory. If you ask about scope, it is the correct way to do it.

Reply:
the original implementation was fine?